api/v1beta1/common.go

```diff
@@ -41,23 +143,39 @@ type WorkflowCommonParameters struct {
 	// functionalities to work properly (e.g.: extraRPMs in Tempest CR, or certain set
 	// of tobiko tests).
 	Privileged *bool `json:"privileged,omitempty"`
-}
 
-type extraConfigmapsMounts struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:MaxLength:=253
-	// The name of an existing config map for mounting.
-	Name string `json:"name"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:="local-storage"
+	// StorageClass used to create any test-operator related PVCs.
+	StorageClass *string `json:"storageClass"`
 
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	// +kubebuilder:validation:Required
-	// Path within the container at which the volume should be mounted.
-	MountPath string `json:"mountPath"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=""
+	// A URL of a container image that should be used by the test-operator for tests execution.
+	ContainerImage *string `json:"containerImage"`
```

Comment:
We are encountering regression. `ContainerImage` is not working in workflow after this change. The issue lays probably here [1]. The type is no longer `string` but `*string`. 

```
---
apiVersion: test.openstack.org/v1beta1
kind: AnsibleTest
metadata:
  namespace: openstack
  name: ansible-tests
spec:
  ...
  containerImage: "quay.io/lpiwowar0/openstack-tempest-all:root"
  ...
  workflow:
    - stepName: "tortuga"
      containerImage: "quay.io/lpiwowar0/openstack-tempest-all:workflow"
      ...
```

[1] https://github.com/openstack-k8s-operators/test-operator/blob/458fbd9e1616c14c6093802f473cbc1c1ebf7367/controllers/ansibletest_controller.go#L352C12-L352C29